doc/source/security/using-amz-kms.rst

```diff
@@ -0,0 +1,135 @@
+.. _using-amz-kms:
+
+================================================================================
+Using the Amazon Key Management Service (AWS KMS)
+================================================================================
+
+This feature is **technical preview** quality.
+
+**Percona Server for MySQL** 8.0.28-20 adds support for the `Amazon Key
+Management Server (AWS KMS) <https://aws.amazon.com/kms/>`__. The AWS server
+generates the keyring material and uses a local file for key storage.
+
+The AWS KMS lets you create and manage cryptographic keys across AWS services. For more information, see the
+`AWS Key Management Service Documentation <https://docs.aws.amazon.com/kms/>`__.
+
+To use the AWS KMS component, you must have the following:
+
+* Have an AWS KMS account.
+* Create a secret access key ID and a secret access key. The access key verifies
+  your identity and the application's identity.
+
+* Create a KMS key ID.
+
+
+
+
+Component installation
+--------------------------------------
+
+Install the AWS KMS component with a manifest. You should only load the
+component with a manifest file. The server uses this manifest file and the
+component consults its configuration file during initialization. Do not load this component with the ``--early-plugin-load`` server option.
```

Comment:
You can't load it with early-plugin-load, as that's for plugins, not components.

Reply:
Removed.